runtime/v1/shim/client/client.go

```diff
@@ -107,6 +109,10 @@ func WithStart(binary, address, daemonAddress, cgroup string, debug bool, exitHa
 			"address": address,
 			"debug":   debug,
 		}).Infof("shim %s started", binary)
+
+		if err := shimv2.WriteAddress(filepath.Join(config.Path, "address"), address); err != nil {
```

Comment:
could we create the same function for this? I think v1 should not import v2 package here. WDYT?

Reply:
Yes, this sounds good

Reply:
updated